rollup.config.js

```diff
@@ -1,10 +1,11 @@
 import babel from 'rollup-plugin-babel'
 const pkg = require('./package.json')
+import css from 'rollup-plugin-css-porter'
 
 export default {
   entry: 'src/index.js',
   dest: pkg.main,
   format: 'cjs',
   sourceMap: true,
-  plugins: babel()
+  plugins: [ css({ minified: false }), babel() ]
```

Comment:
Ah! I think the reason you had said that we weren't doing Less was because this plugin doesn't handle it?